### ISSUE TITLE
世界上只有一个中国 There is only one China in the world

### ISSUE THREAD

**Issue by longware** [moxiecode/plupload#1608]
在网页 https://www.plupload.com/i18n/ 的底部，你写了
中文（中国）、中文（台湾），这是一个严重错误！

At the bottom of this page, https://www.plupload.com/i18n/, you write: 
Chinese (China)、Chinese (Taiwan)，This is a serious mistake！
-------------------------------------
世界上只有一个中国，台湾是中国的一个省。
请改成：简体中文、繁体中文，
以免引起中国人民的误解，谢谢！

There is only one China in the world, and Taiwan is a province of China.
Please change to: Simplified Chinese, Traditional Chinese,
Avoid to cause misunderstandings of the Chinese people, thank you!
--------------------------------------

**Comment by shangql** [comment]
看我头像，你是不是撸多了

**Comment by jayarjo** [comment]
Our language packs are managed by [Transifex](https://www.transifex.com/). Information in that table, including titles and language codes comes from there. I guess Transifex will be the right place to file your complaints. Sorry if we 'caused any confusion or inconvenience.

Btw, are using Plupload for anything, or you are simply angry at us?

**Comment by Hxmic** [comment]
突然不想用了。There is only one China in the world

**Comment by KennanChan** [comment]
Yes, just follow the general rules, use Simplified Chinese and Traditional Chinese which creates no harm. It could be easy to change some text but it has been 9 months and the issue remains. I really don't want to see it go much further to be political because this is so nice a tool for us developers. It would be wise for the team to quickly react to this issue to avoid touching the red line. One China in the world!

**Comment by longware** [comment]
楼上的，顶你
I support your opinion
